manifests/base/deployment.yaml

```diff
@@ -39,9 +39,19 @@ spec:
           protocol: TCP
         readinessProbe:
           httpGet:
-            path: /healthz
+            path: /readyz
             port: main-port
             scheme: HTTPS
+          initialDelaySeconds: 10
+          periodSeconds: 20
+          failureThreshold: 3
+        livenessProbe:
+          httpGet:
+            path: /livez
+            port: main-port
+            scheme: HTTPS
+          periodSeconds: 20
+          failureThreshold: 3
```

Comment:
As the default resolution is 60 sec, using this config so there can be a successful scrape within 20*3 seconds

Reply:
This is independent, this tells that probe must fail 3 times every 20s, but to probe fail scraping  halted for 60s (default).
What your changing here is time to detecting fail, this definition will require 120s of halted scraping for container to be restarted (20sx3 + 60s)